seed/python-sdk/oauth-client-credentials/token-override/src/seed/client.py

```diff
@@ -0,0 +1,323 @@
+# This file was auto-generated by Fern from our API Definition.
+
+from __future__ import annotations
+
+import typing
+
+import httpx
+from .core.api_error import ApiError
+from .core.client_wrapper import AsyncClientWrapper, SyncClientWrapper
+from .core.oauth_token_provider import AsyncOAuthTokenProvider, OAuthTokenProvider
+
+if typing.TYPE_CHECKING:
+    from .auth.client import AsyncAuthClient, AuthClient
+    from .nested.client import AsyncNestedClient, NestedClient
+    from .nested_no_auth.client import AsyncNestedNoAuthClient, NestedNoAuthClient
+    from .simple.client import AsyncSimpleClient, SimpleClient
+
+
+class SeedOauthClientCredentials:
+    """
+    Use this class to access the different functions within the SDK. You can instantiate any number of clients with different configuration that will propagate to these functions.
+
+    Parameters
+    ----------
+    base_url : str
+        The base url to use for requests from the client.
+
+    client_id : typing.Optional[str]
+    client_secret : typing.Optional[str]
+    token : typing.Optional[str]
```

Comment:
for this docstring, I think it would be nicer to document the overloads instead. Or maybe document them in a unified manner. E.g. "you can pass in client_id + client_secret as required strings OR you can pass in token